creator-node/src/services/initAudiusLibs.js

```diff
@@ -0,0 +1,84 @@
+const { libs: AudiusLibs } = require('@audius/sdk')
+
+const config = require('../config')
+const { logger: genericLogger } = require('./logging')
+
+/**
+ * Creates, initializes, and returns an audiusLibs instance
+ *
+ * Configures dataWeb3 to be internal to libs, logged in with delegatePrivateKey in order to write chain TX
+ */
```

Comment:
this function is moved from `_initAudiusLibs()` in `serviceRegistry` with following changes:
- all config variables are defined at the top
- thats it